### PR TITLE
Run3-gex14 Remove some overlap in Run3 geometry

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Remove some overlaps coming in the PPS part of Run3 geometry

#### PR validation:

Tested using the G4 overlap tool

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Configuration/Geometry/g4n.log [hunk 1]
```diff
@@ -0,0 +1,7 @@
+----- Begin Fatal Exception 24-Sep-2020 19:32:24 CEST-----------------------
```
Comment (vargasa) [inline Configuration/Geometry/g4n.log:1]: Can this file be removed?

Comment (bsunanda) [reply Configuration/Geometry/g4n.log]: Sorry - I missed it. I did a "git add ." without realizing the consequence

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Configuration/Geometry/g4n.log (hunk 1 above)
Comment (makortel) [reply Configuration/Geometry/g4n.log]: Could the two commits be squashed together to avoid the file appearing in the git history (even if it is small)?

**Comment by cvuosalo** [comment]
+1

**Comment by kpedro88** [comment]
+upgrade

**Comment by civanch** [comment]
+1

**Comment by qliphy** [comment]
+1

**Comment by makortel** [comment]
Apparently this PR caused two unit tests in DetectorDescription/DDCMS to fail
```
==== Let's go to #45
>>> 5 level: PixelBarrelSuppBox is a Box
Full path to it is world_volume/OCMS_1/CMSE_1/Tracker_1/PixelBarrel_1/PixelBarrelSuppBox_2
    copy Nos: 1, 1, 1, 1, 2, 
    levels  : 0, 0, 6, 2, 2, 
F

DDFilteredView.goto.cppunit.cc:68:Assertion
Test name: testDDFilteredViewGoTo::checkFilteredView
assertion failed
- Expression: it == refPos_[i++]

Failures !!!
Run: 1   Failure total: 1   Failures: 1   Errors: 0

---> test testDD4hepFilteredViewGoTo had ERRORS
```
and
```
Go to #45
5 level: PixelBarrelSuppBox is a Box
Full path to it is world_volume/OCMS_1/CMSE_1/Tracker_1/PixelBarrel_1/PixelBarrelSuppBox_2
2, 1, 1, 1, 1, 
0, 0, 6, 2, 2, 
0, 0, 6, F

DDFilteredView.level.cppunit.cc:93:Assertion
Test name: testDDFilteredViewLevel::checkFilteredView
assertion failed
- Expression: it == refPos_[count++]

Failures !!!
Run: 1   Failure total: 1   Failures: 1   Errors: 0

---> test testDD4hepFilteredViewLevel had ERRORS
```

For more information see http://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc820/CMSSW_11_2_X_2020-09-27-0000/unitTestLogs/DetectorDescription/DDCMS

**Comment by bsunanda** [comment]
@markotel This PR has nothing to do with that failure. PixelBarrelSuppBox belongs to Tracker and this PR deals with PPS related xml's> One needs to investigate the failures independently

**Comment by silviodonato** [comment]
@bsunanda these errors appeared since `CMSSW_11_2_X_2020-09-27-0000` and this PR is the only PR merged between `CMSSW_11_2_X_2020-09-26-1100` and `CMSSW_11_2_X_2020-09-27-0000`, so they seem related to this PR. Anyway I will double check this manually.

**Comment by silviodonato** [comment]
@bsunanda 
I tried in `CMSSW_11_2_X_2020-09-27-2300`
Baseline: error in `testDD4hepFilteredViewLevel` and `testDD4hepFilteredViewGoTo`
I tried again after having copied 
`cp Geometry/CMSCommonData/data/extend/v2/cmsextent.xml Geometry/CMSCommonData/data/extend/v3/cmsextent.xml`
and I got
```
---> test testDD4hepFilteredViewLevel succeeded
---> test testDD4hepFilteredViewGoTo succeeded 
```

**Comment by silviodonato** [comment]
@bsunanda I investigated a bit the issue. 
You can compare the log from [CMSSW_11_2_X_2020-09-26-1100](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/raw/slc7_amd64_gcc820/CMSSW_11_2_X_2020-09-26-1100/unitTestLogs/DetectorDescription/DDCMS) and from [CMSSW_11_2_X_2020-09-27-0000](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/raw/slc7_amd64_gcc820/CMSSW_11_2_X_2020-09-27-0000/unitTestLogs/DetectorDescription/DDCMS).
In particular there is a change in the level numbers
from (`CMSSW_11_2_X_2020-09-26-1100`)
```
#1: >>> 1 level: OCMS is a Box
Full path to it is world_volume/OCMS_1
    copy Nos: 1, 
    levels  : 0, 
#2: >>> 2 level: CMSE is a Box
Full path to it is world_volume/OCMS_1/CMSE_1
    copy Nos: 1, 1, 
    levels  : 0, 0, 
#3: >>> 3 level: CMStoZDC is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/CMStoZDC_1
    copy Nos: 1, 1, 1, 
    levels  : 0, 0, 0, 
#4: >>> 3 level: CMStoZDC is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/CMStoZDC_2
    copy Nos: 1, 1, 2, 
    levels  : 0, 0, 1, 
#5: >>> 3 level: ZDCtoFP420 is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/ZDCtoFP420_1
    copy Nos: 1, 1, 1, 
    levels  : 0, 0, 2, 
#6: >>> 3 level: ZDCtoFP420 is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/ZDCtoFP420_2
    copy Nos: 1, 1, 2, 
    levels  : 0, 0, 3, 
#7: >>> 3 level: Tracker is a Polycone
Full path to it is world_volume/OCMS_1/CMSE_1/Tracker_1
    copy Nos: 1, 1, 1, 
    levels  : 0, 0, 4, 
```
to (`CMSSW_11_2_X_2020-09-27-0000`)
```
#1: >>> 1 level: OCMS is a Box
Full path to it is world_volume/OCMS_1
    copy Nos: 1, 
    levels  : 0, 
#2: >>> 2 level: CMSE is a Box
Full path to it is world_volume/OCMS_1/CMSE_1
    copy Nos: 1, 1, 
    levels  : 0, 0, 
#3: >>> 3 level: CMStoZDC is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/CMStoZDC_1
    copy Nos: 1, 1, 1, 
    levels  : 0, 0, 0, 
#4: >>> 3 level: CMStoZDC is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/CMStoZDC_2
    copy Nos: 1, 1, 2, 
    levels  : 0, 0, 1, 
#5: >>> 3 level: ZDCtoFP4201 is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/ZDCtoFP4201_1
    copy Nos: 1, 1, 1, 
    levels  : 0, 0, 2, 
#6: >>> 3 level: ZDCtoFP4201 is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/ZDCtoFP4201_2
    copy Nos: 1, 1, 2, 
    levels  : 0, 0, 3, 
#7: >>> 3 level: ZDCtoFP4202 is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/ZDCtoFP4202_1
    copy Nos: 1, 1, 1, 
    levels  : 0, 0, 4, 
#8: >>> 3 level: ZDCtoFP4202 is a Tube
Full path to it is world_volume/OCMS_1/CMSE_1/ZDCtoFP4202_2
    copy Nos: 1, 1, 2, 
    levels  : 0, 0, 5, 
#9: >>> 3 level: Tracker is a Polycone
Full path to it is world_volume/OCMS_1/CMSE_1/Tracker_1
    copy Nos: 1, 1, 1, 
    levels  : 0, 0, 6, 
```

If we remove two out of `ZDCtoFP4201_1`, `ZDCtoFP4201_2`, `ZDCtoFP4202_1`, `ZDCtoFP4202_2`, the level numbers do not change (eg. `levels  : 0, 0, 4, ` for Tracker) and then the tests pass.
I'm not sure if this means there is an error in the geometry or in the unit test.

**Comment by bsunanda** [comment]
This is surely error in the unit test. It cannot test on number of children in CMSE or on absolute position of a given child in the mother. It is better to inhibit the unit test for the time being till a concrete test is found.
________________________________

**Comment by bsunanda** [comment]
It is better to remove levels from the unit test - can you let me know the name of the unit test?

**Comment by makortel** [comment]
@ianna already made a fix in https://github.com/cms-sw/cmssw/pull/31605.

**Pull request merged**